website/docs/connect-data/reference/appsmith-ai.md

```diff
@@ -0,0 +1,278 @@
+# Appsmith AI
+
+This page provides information on creating queries with Appsmith AI, which allows you to configure applications with advanced AI features such as text generation, image classification, and sentiment analysis without the need for any API keys or datasource authentication.
+
+
+
+
+<ZoomImage
+  src="/img/appsmith-ai-2.png" 
+  alt="Appsmith AI"
+  caption="Appsmith AI query"
+/>
+
+
+## Query Appsmith AI
+
+The following section is a reference guide that provides a description of the available commands with their parameters to create Appsmith AI queries.
+
+ You can try the examples provided in the query editor to understand and experiment with the capabilities of Appsmith AI.
+
+### Generate text 
+
+
+
+The Generate Text action allows you to generate text based on specific prompts.
+
+
+<dd>
+
+#### Prompt
+
+
+
+This field allows you to input prompts for text generation. You can also add data from queries or JSObjects using mustache bindings `{{}}`.
+
+For example, "Write a creative product description for an electric car named RevaX, with the keywords- fast charging, 200 miles per single charge, compact, eco-friendly, economical. This is targeted towards a climate concious tech/EV enthusiast who enjoys driving. Also describe the benefits of this product in less than 80 words."
+
+
+
+</dd>
+
+
+
+### Classify text
+
+
+
+This action allows you to analyze and categorize text based on its content. You can gain insights into the nature of the provided text, making it a valuable for text classification tasks in your applications.
+
+
+
+<dd>
+
+#### Input
+
+
+
+This field enables you to input prompts for text classification. You can provide either a simple text or an array of text, such as chat details, for classification. Additionally, you can add data from queries or JSObjects using mustache bindings `{{}}`.
+
+For example, you can input a user's product review, like:
+
+"Received my laptop stand from Macazon after a significant delay in delivery, which was a bit disappointing. Upon unboxing, I noticed the build seemed sturdy, but unfortunately, the stand doesn't function as expected. It's supposed to be adjustable, but the mechanism is quite stiff, making it challenging to change angles or heights."
+
+</dd>
+
+<dd>
+
+#### Labels
+
+
+
+Labels refer to predefined categories or classes assigned to the analyzed text. These labels are essential for organizing and identifying the nature of the text content. 
+
+For example, if you are classifying product reviews, labels could include categories such as "Positive," "Negative," or "Neutral."
+
+</dd>
+
+<dd>
+
+#### Additional Instructions
```

Comment:
Let users know that they can add descriptions for labels in the additional instructions. 

eg: 
Labels: Urgent, High priority, Low

Additional Instructions: Use the following labels descriptions: 
Urgent: Issues that prevent a customer from using the product
High priority: Issues that are nasty but still makes the product usable
Low priority: Issues that can be lived with